trustymail/trustymail.py

```diff
@@ -415,15 +406,33 @@ def scan(domain_name, timeout, smtp_timeout, smtp_localhost, smtp_ports, smtp_ca
     return domain
 
 
-def handle_error(prefix, domain, error):
-    if hasattr(error, "message"):
-        if "NXDOMAIN" in error.message and prefix != "[DMARC]":
+def handle_error(prefix, domain, error, syntax_error=False):
```

Comment:
Can we add a docstring here to explain what the goal / overall effect of this custom handling is?

Reply:
Yes.  I intend to add docstrings everywhere in `trustymail` and `pshtt` at some point, but I'll go ahead and add one for this function.